build/Dockerfile

```
@@ -0,0 +1,24 @@
+FROM golang:1.19-alpine as builder
+
+ENV GO111MODULE=on
+ENV CGO_ENABLED=0
```

question (non-blocking): is there a reason we keep these ENV variables on separate lines and layers?

Maybe we keep them on one line and save ourselves a layer:
```suggestion
ENV GO111MODULE=on CGO_ENABLED=0
```

---

it's not relevant for the final image, since we have a multistage build
but it might save 1s during the build process

---

Ah, you’re right, since it’s multi-stage. Thanks for clarifying!